experimental/Makefile

```diff
@@ -0,0 +1,73 @@
+SHELL := /bin/bash
+.ONESHELL:
+.SHELLFLAGS := -eu -o pipefail -c
+.DELETE_ON_ERROR:
+MAKEFLAGS += --warn-undefined-variables
+MAKEFLAGS += --no-builtin-rules
+
+# Avoid tabs
+# brew install make --with-default-names
+# brew install homebrew/dupes/make --with-default-names
```

Comment:
czo to sa te taby?

Reply:
https://www.gnu.org/software/make/manual/html_node/Special-Variables.html

Sometimes a single space can mess up the makefile. And good luck debugging :/
This swaps `tabs` for `>` as `RECIPEPREFIX` to help fight that

Reply:
wrzucilem te flagi; spoko som https://github.com/awinecki/magicfile/commit/d150f69e1eae61b49e07c30bb8dcbfa8b26063ed

ale z tymi tabami to olewam temat. To wyglada na mocno macos-specific, a chcialbym zeby to dzialalo na linuxach tez.

Reply:
Ok